gui/src/components/Panel.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+  import IconButton from './IconButton.svelte';
+  import BackSVG from './mono/leftarrow.svelte';
+  import * as router from 'svelte-spa-router';
+
+  export let title: string | undefined = undefined;
+  export let backRoute: string | undefined = undefined;
+</script>
+
+<div class="panel">
+  {#if $$slots.actions || title !== undefined || backRoute !== undefined}
```

Comment:
If you take the above advice, this becomes `title !== null`. However, a generally more robust check might be `title != null` which checks for == null || == undefined.

Also, while I generally prefer explicit boolean checks rather than relying on "truthiness", it's actually really common and convenient in UI code. So you could even just shorten this to `|| title || backRoute` and drop the null/undefined checks altogether. In that case, you could also change the export to `export let title: string = '';`